### PR TITLE
Fix integrated yaw usage

### PR DESCRIPTION
**Pull request by Asizon**
Allow manual pid change when integrated yaw is enabled.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — DISMISSED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
After testing I found we have to press the Enable button twice.

Moving the code to pid_tuning.js solves it:

```
        $('input[id="useIntegratedYaw"]').change(function() {
            const checked = $(this).is(':checked');
            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
                this.sliderPidsMode = 0;
                $('#pid_main .pid_data input').prop('disabled', false);
            }
            $('#pidTuningIntegratedYawCaution').toggle(checked);
        }).change();

```

**Review by limonspb — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by ctzsnooze** [comment]
In my testing, the PR allows me to edit manually edit PID values, when integrated yaw is enabled, but if I save the Profile, then my changes are lost and I cannot edit the values anymore.

**Comment by haslinghuis** [comment]
@ctzsnooze Can't reproduce as I've tested

**Comment by Asizon** [comment]
Yes, we need to take a look into this issue. Ithink sliders are not changing correctly from RPY to off.

File: src/js/tabs/pid_tuning.js
Comment (haslinghuis) [inline src/js/tabs/pid_tuning.js:551]:
```suggestion
                TuningSliders.sliderPidsMode = 0;
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Comment by haslinghuis** [comment]
Sorry @Asizon I made a typo in my first suggestion. Now I have more time I had a fresh look and found you should move the fix to around line 1936 as this only applies to sliders. We can leave the code around 550 untouched:
```
$('input[id="useIntegratedYaw"]').change(() => {
    if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
        TuningSliders.sliderPidsMode = 0;
    }
    TuningSliders.updatePidSlidersDisplay();
}
```

The typo was `this.sliderPidsMode` instead of `TuningSliders.sliderPidsMode`.

**Comment by klutvott123** [comment]
So enabling integrated yaw should disable sliders for RPY, not just Y?

**Comment by haslinghuis** [comment]
Hmm you suggest setting slider to RP mode instead to OFF? Makes sense to me. Testing now.

**Comment by klutvott123** [comment]
I suggest setting it to RP IF it's currently set to RPY

**Comment by haslinghuis** [comment]
@Asizon 

Worked out the changes needed @klutvott123 suggested:

[diff2.txt](https://github.com/betaflight/betaflight-configurator/files/7629756/diff2.txt)

**Comment by ctzsnooze** [comment]
@Asizon - do you have time to update this PR and test it?  We need to resolve this for 10.8 RC1 as soon as possible.

**Review by haslinghuis — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Comment by ctzsnooze** [comment]
Tested, works very effectively.

**Review by limonspb — APPROVED** [review]

**Pull request merged**
